test/restart-task-test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var expect = require('expect.js');
+var sinon = require('sinon');
+var cron = require('../src/node-cron');
+
+describe('restarting a task', function() {
+  beforeEach(function() {
+    this.clock = sinon.useFakeTimers();
+  });
+
+  afterEach(function() {
```

Comment:
There isn't the .jshintrc in your branch, you should merge master into your branch to fix it.